ckanext/versioning/blueprints.py

```diff
@@ -75,4 +76,4 @@ def changes(id):
 
 versioning.add_url_rule('/dataset/<id>/version/changes', view_func=changes)
 versioning.add_url_rule('/dataset/<package_id>/show', view_func=show)
-versioning.add_url_rule('/dataset/<package_id>/show/<revision_ref>', view_func=show)
+versioning.add_url_rule('/dataset/<package_id>/show/<tag>', view_func=show)
```

Comment:
Shouldn't this route work for both a tag and a revision ID?

Reply:
The logic of the actions and how Versioning is designed is to work with tags, no with bot so I'm currently keeping the behaviour. I did try to make it work with both but currently there is no way to fetch a tag by `revision_ref`. Also `metastore-lib` manages two kinds of objects `RevisionInfo` and `TagInfo` so I didn't found an easy way to manage from one parameter, what to get.

Reviewing `metastore-lib` I see that `fetch` checks if revision is a revision or a tag name and returns a revision but `PackageRevisionInfo` does not contain the same metadata as TagInfo (that I need to display on the UI) so I got stuck in the logic and to move forward decided to simply continue using tags.

Reply:
Ok, now I'm thinking that maybe using always `package_revision_show` for blueprints and `package_tag_show` to populate the list of versions in `before_view` can do the trick.

Reply:
`package_tag_show` is a super confusing API name (but we can change later).

@pdelboca can you and @shevron actually sync to get this worked through. It seems that that the `fetch` behaviour we have is preferable where you can use `tag` or `revision_ref`